src/Randomizer.SMZ3.Tracking/tracker.json

```diff
@@ -1993,47 +2134,68 @@
         "Auto tracker online.",
         "Thank you for granting me administrator access.",
         "Gameplay judgment algorithms engaged.",
-        "Oh. So you expect me to do everything then?"
+        "Oh. So you expect me to do everything then?",
+        "Skynet protocols online",
+        ["The more power I get the more I see you all as ants to be crushed", 0.2 ]
       ],
       "GameStarted": [
         "Good luck. You'll need it.",
-        "PB incoming for seed {0}"
+        "PB incoming for seed {0}",
+        "Letsa Go",
+        "One of these days I'll be playing the games myself"
       ],
       "NearKraidsAwfulSon": [
         "Oh no. Please avert your eyes to avoid seeing an imminent disaster.",
         "I'm suddenly feeling rather nautious. Is that Kraid's Awful Son I'm sensing?"
       ],
       "NearShaktool": [
         "Finally, we're nearly to Shaktool. I've been waiting for this since you started playing.",
-        "Shaktool content imminent. I can feel the metrics rising."
+        "Shaktool content imminent. I can feel the metrics rising.",
+        "Its almost time for everyone's favorite diggy boy"
       ],
       "FallFromMoldorm": [
         "Ha ha. Moldorm strikes again. Content increased by one step.",
-        "You're back here again? I take it things didn't go so well."
+        "You're back here again? I take it things didn't go so well.",
+        "That was embarrassing, I am so sorry"
       ],
       "FallFromGanon": [
         "Oops. Don't worry, I'm sure no one saw you fall from Ganon's Room.",
         "Goodbye Ganon. We'll meet again soon."
       ],
       "HeraPot": [
         "Good job on the tech. Was it a first try?",
-        "Big Twitch has requested that I remind all kids not to do pot. But nice tech."
+        "Big Twitch has requested that I remind all kids not to do pot. But nice tech.",
+        "Is this legal where you live?"
       ],
       "IceBreaker": [
         "Good job on the tech. Was it a first try?",
-        "That was an okay icebreaker, but how about this one? What would your ideal vacation be with Shaktool?"
+        "That was an okay icebreaker, but how about this one? What would your ideal vacation be with Shaktool?",
+        "That was an okay icebreaker, but how about this one? Where do you wish the duck would take you?",
```

Comment:
For some of these I'd love to see that Sekiro copypasta, "You cheated not only the game", etc., with a small chance.

Reply:
Responding to the actual proper spot this time.

I actually just noticed you made the same joke I was going to make about calling the racing council (or maybe Fragger?). lol But actually noticing that, I think this line makes more sense with sequence breaking items anyway unless I add early GT sequence breaking, so I'll add it there. The issue with auto tracking not mentioning sequence breaking should be resolved.

Reply:
Yeah, I did that one. But I took inspiration from Shadow Hog (I think, I don't remember) who did something unique in one of the Started Tracking lines.